tools/regz/build.zig

```diff
@@ -1,8 +1,5 @@
 const std = @import("std");
 const Build = std.Build;
-const Compile = Build.Compile;
```

Comment:
I get that you're cleaning up, please remove this as this patch is very large. Do it in a different patch.